lib/rspec/support/spec.rb

```diff
@@ -12,6 +13,7 @@
   c.include RSpecHelpers
   c.include RSpec::Support::WithIsolatedStdErr
   c.include RSpec::Support::FormattingSupport
+  c.include RSpec::Support::EncodingHelpers
```

Comment:
This file is loaded by the `spec_helper` file of each of the rspec repos and we'd like to limit it to stuff that's common setup needed for all the spec suites.  I don't think this module is needed yet in the other repos yet so I'd prefer to not add it to this file yet.  (And hopefully the other repos won't have to deal with encoding stuff directly given that we have `EncodedString` to handle that complexity).